test/networkd-test.py

```diff
@@ -96,10 +96,17 @@ def write_network_dropin(self, unit_name, dropin_name, contents):
         dropin_path = os.path.join(dropin_dir, "%s.conf" % dropin_name)
 
         os.makedirs(dropin_dir, exist_ok=True)
+        self.addCleanup(os.rmdir, dropin_dir)
         with open(dropin_path, 'w') as dropin:
             dropin.write(contents)
         self.addCleanup(os.remove, dropin_path)
 
+    def read_attr(self, link, attribute):
+        """Read a link attributed from the sysfs."""
+        self.assert_link_states(**{link:'managed'})
+        with open('/'.join(['/sys/class/net', link, attribute])) as f:
```

Comment:
But I cannot do that, can I? I don't want to pass kargs argument named link, but the value of link argument passed to the function. E.g. when one calls `self.read_attr('eth1', foo)`, i want to call `self.assert_link_states(eth1='managed')`.